contracts/controller/src/liquidation.rs

```diff
@@ -77,9 +77,11 @@ impl Contract {
     ) -> WBalance {
         WBalance::from(
             self.get_liquidation_incentive().0
-                * liquidation_amount.0
-                * self.prices.get(&borrowing_dtoken).unwrap().value.0
-                / (self.prices.get(&collateral_dtoken).unwrap().value.0 * RATIO_DECIMALS.0),
+                * Balance::from(
+                    liquidation_amount * self.prices.get(&borrowing_dtoken).unwrap().value
+                        / (self.prices.get(&collateral_dtoken).unwrap().value),
+                )
+                * RATIO_DECIMALS.0,
```

Comment:
@dmytro-shash

previously here was 
```
liquidation_amount.0 * self.prices.get(&borrowing_dtoken).unwrap().value.0 / (self.prices.get(&collateral_dtoken).unwrap().value.0 * RATIO_DECIMALS.0),
```
and now
```
liquidation_amount * self.prices.get(&borrowing_dtoken).unwrap().value / (self.prices.get(&collateral_dtoken).unwrap().value) * RATIO_DECIMALS.0
```

Reply:
done